packages/components/psammead-figure/README.md

```diff
@@ -0,0 +1,51 @@
+# psammead-figure &middot; [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://bbc-news.github.io/psammead/?selectedKind=Figure) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-figure.svg)](https://www.npmjs.com/package/@bbc/psammead-figure) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/BBC-News/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+The `psammead-figure` component is a styled `<figure>` element.
+
+## When to use this component
+
+Use this component when a `<figure>` element, which represents self-contained content that frequently has a caption, is required. Figures usually wrap an image.
```

Comment:
```suggestion
Use the Figure component to associate a caption with an image.
```

Reply:
I think your suggestion is too specific. That's because a figure doesn't technically need a caption, though it frequently has one. Also, it's not necessary to use an image with a figure. A code block or data viz for example, could be in a figure
https://www.w3.org/TR/html53/grouping-content.html#the-figure-element

Reply:
Fair enough 👍 it just reads quite clunky to me.